main.go

```diff
@@ -37,7 +37,6 @@ func main() {
 	utils.Version = version
 	if binaryToDocker := os.Getenv("BINARY_TO_DOCKER"); binaryToDocker != "true" {
 		fmt.Println(logo, " ")
-		fmt.Printf("version: %v\n\n", version)
```

Comment:
why did you remove this?

Reply:
Refrence: https://github.com/keploy/keploy/issues/1488

Reply:
I don't see any comments on that issue, did you discuss this with any organisation member before implementing? We usually print the version along with logo because whenever someone faces a problem with Keploy, we ask for the logs, and there we can check what specific version is causing the issue.

Reply:
Yes this was prior discussed with @charankamarapu , It was supposed to be implemented as a part of #1376, but was asked to be removed as some discussions needed to be done before the implementation. 

Can we have a discussion over this on slack and decide whats the best course of action regarding the version message?

Reply:
> What needs to be discussed is the delay that will happen(~2-3s) due to an Http request being made everytime keploy is run
> 
> There are two possible solutions:
> 
> * The update message is shown at the end of execution of the command
> * The update message is shown only once a day.
> 
> Let us discuss on how this can be implemented.


This also needs to be discussed with the community

Reply:
I had a chat with him and I think we are now supposed to run keploy --version everytime we start Keploy. Is that correct? Please let me know when youa add that change

Reply:
Updated, Thank you for resolving

Reply:
As it turns out, Keploy automatically adds in a --version flag upon initialisation, which was customised in #1376. Instead of running keploy with --version command everytime, i think its best we just update the message to 
```Keploy version: 2-xyz```


Let me know if this is an optimal approach

Reply:
I think it was finaliased that the version flag should be run everytime because showing the version like this is not a good pracice so I think we should stick to that.